libraries/chain/asset_evaluator.cpp

```diff
@@ -83,11 +83,6 @@ void_result asset_create_evaluator::do_evaluate( const asset_create_operation& o
          FC_ASSERT( asset_symbol_itr->issuer == op.issuer, "Asset ${s} may only be created by issuer of ${p}, ${i}",
```

Comment:
Please replace the TODO in line 39 with a comment why this has to stay, to simplify future cleanups.

Reply:
Cleaned up code and commented as part of #1743